float32 torsion_modulus
---
bool success
